make/test.mk

```diff
@@ -106,23 +106,23 @@ endif
 # e2e test in dev mode
 #-------------------------------------------------------------------------------
 
-.PHONY: e2e-cleanup
+.PHONY: e2e-cleanup-local
```

Comment:
[e2e-cleanup-local](https://github.com/redhat-developer/devconsole-operator/pull/120/files#diff-57dae1b5b313e109d7ebdfd77a500c5cR109) is similar to [e2e-cleanup](https://github.com/redhat-developer/devconsole-operator/pull/120/files#diff-57dae1b5b313e109d7ebdfd77a500c5cR52) . Can't we remove one of them?

Reply:
@DhritiShikhar I will address this in a separate PR.

Reply:
@baijum okay but what is the need of this change then?

Reply:
@DhritiShikhar To avoid ambiguous Makefile target names.  Same names are repeated. Your question is more the content within the target. That we can address separately.